vision/detect/detect_test.go

```diff
@@ -66,3 +72,62 @@ func TestDetect(t *testing.T) {
 		}
 	}
 }
+
+func TestDetectAsyncDocument(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	ctx := context.Background()
+
+	// Create a temporary bucket
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	bucketName := fmt.Sprintf("%s-golang-samples-%d", tc.ProjectID, time.Now().Unix())
```

Comment:
It can cause problems to rely on `time.Now().Unix()` to be unique. Could this reuse a single bucket (with the projectID in the bucket name)?